### DISCUSSION TITLE
Creating Issue

### DISCUSSION THREAD

**Discussion by Islamation** [community/community#113258] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I create new list (after pressing Create) it kept on "Creating" and fixed on it

**Comment by faisalbasra** [comment]
This is same issue for me and following is screenshot. 
![Create List](https://github.com/community/community/assets/1409742/dbc6edca-0595-4c22-95dc-b986e4f121a7)

**Reply by MusaMahmood** [reply]
Observation: If you refresh the page, it appears the operation does complete correctly. The UI just hangs on creation.

**Reply by 00676** [reply]
**thanks![IMG-20240706-WA0016.jpg](https://github.com/community/community/assets/158535979/9416b1d3-36c6-4237-b65b-2c6747f21c43)

**

**Comment by 212sheyik** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> When I create new list (after pressing Create) it kept on "Creating" and fixed on it

**Comment by Lovebugpaul50** [comment]
Love it

**Comment by Lovebugpaul50** [comment]
Love it

**Comment by Lovebugpaul50** [comment]
LOVE IT

**Reply by 212sheyik** [reply]
- [ ] []()~~**@ **~~

**Comment by Roddy8720** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> When I create new list (after pressing Create) it kept on "Creating" and fixed on it

**Comment by vegeta321311** [comment]
- [ ] @davyus.eth~~~~
- [ ] @Sdavyus~~~~
- [ ] @vegeta321311~~~~
# @simard550~~~~
- [ ] @davyussigmar~~~~
- [ ] @DavidSimard~~~~

**Comment by 1alexbc1** [comment]
Ad
